net/local/local_sendpacket.c

```diff
@@ -102,6 +103,114 @@ static int local_fifo_write(FAR struct file *filep, FAR const uint8_t *buf,
   return nwritten > 0 ? nwritten : ret;
 }
 
+/****************************************************************************
+ * Name: local_sendctl
+ *
+ * Description:
+ *   Handle the socket message conntrol field
+ *
+ * Input Parameters:
+ *   conn     Local connection instance
+ *   msg      Message to send
+ *
+ * Returned Value:
+ *  On any failure, a negated errno value is returned
+ *
+ ****************************************************************************/
+
+#ifdef CONFIG_NET_LOCAL_SCM
+static int local_sendctl(FAR struct local_conn_s *conn,
```

Comment:
call local_sendctl in local_sendmsg, so we don't need modify local_sendctl prototype